src/routes/Home.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Books from '../components/Books';
 import AddBooks from '../components/AddBooks';
+import '../index.css';
 
 const Home = () => (
```

Comment:
- [ ] You have still not implemented the change from the previous reviewer. Kindly compare the designs and stick to it as much as possible

  - The percent score should move closer to the right, next to the Update Progress button
  - The Book form should be made wider to cover the entire length of the container just like the container of the book.

  Your Implementation | Zeplin Design
  :---:|:---:
  ![image](https://github.com/Dorcas126/Book-store-project/assets/36542986/669b93d4-c748-416e-b802-a0b4fa48a43b) | ![image](https://github.com/Dorcas126/Book-store-project/assets/36542986/fc662759-9835-4fd5-b6ef-7306d0357310)